shotgun_api3/shotgun.py

```diff
@@ -693,12 +693,34 @@ def info(self):
         Get API-related metadata from the Shotgun server.
 
         >>> sg.info()
-        {'full_version': [6, 3, 15, 0], 'version': [6, 3, 15], ...}
+        {'full_version': [8, 2, 1, 0], 'version': [8, 2, 1], 'user_authentication_method': 'default', 'unified_login_flow_enabled': True, ...}
+
+        Tokens and values
+        -----------------
+
+        ::
+
+            Token                       Value
+            --------                    ---------
+            full_version                An ordered array of the full Shotgun version.
+                                        [major, minor, patch, hotfix]
+            version                     An ordered array of the Shotgun version.
+                                        [major, minor, patch]
+            user_authentication_method  Indicate the authentication method used by Shotgun.
+                                        Will be one of the following value:
+                                            default: regular username/password.
+                                            ldap:    username/password from the company's LDAP.
+                                            saml2:   SSO used, over SAML2.
+            unified_login_flow_enabled  Indicate if the /auth/renew endpoint is enabled.
```

Comment:
This part i find confusing. Rather than explaining the underlying implementation, should we explain what it means to the end user! Do we have some technical documentation explaining this that we could refer to? What does it mean for API users? When is this flag useful for someone using the API?

Reply:
Actually... this token appeared to show the value of a hidden pref... Hidden pref that has now disappeared since the Unified Login Flow is now present in any 8.* builds.

I'm currently removing the need for that token in RV and in my SG Desktop/Toolkit work, and spoken to the Shotgun Create team.

As it will disappear from Shotgun soon, I have removed the documentation on it.